go/store/prolly/tree/z_encoding.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package index
```

Comment:
It got moved along with the `PutField` and `GetField` functions because `PutField` depends on `ZCell` for writing geometry values.

Those functions got moved because the improved `OutputProllyNode`, which is used for printing prolly tree nodes for debugging, now depends on `GetField` in order to convert the rows into human-readable values.

In the commit description I acknowledge that `prolly/tree` isn't the best place for them and it might make more sense to move them to either `store/val` or to their own package. But if they're only used for indexes as you say it might make the most sense to keep them `sqle/index`, and then find another way to break the dependency cycle.